src/main/java/org/haffson/adventofcode/days/day01/Day01.java

```diff
@@ -41,21 +40,94 @@ public int getDay() {
 
     @Override
     public String firstPart() {
-        return "Part 1 - Frequency: " + calculateFrequency();
+        return "Part 1: " + calculatePart1();
     }
 
     @Override
     public String secondPart() {
-        return null;
+        return "Part 2: " + calculatePart2();
     }
 
     /**
      * Primary method for Day 1, Part 1.
-     * Calculates the final frequency as the sum of all frequencies.
+     * Calculates the Report
+     *
+     * @return Returns the product of the multiplication of two numbers that add up to 2020.
+     */
+    private int calculatePart1() {
+        int[] entries = this.findTwoEntriesThatSumToX(this.numbers, 2020);
```

Comment:
The `this` keyword is not really needed in Java, but it also does not do any harm.